src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LoweredDynamicOperationFactory.cs

```diff
@@ -30,7 +30,7 @@ internal LoweredDynamicOperationFactory(SyntheticBoundNodeFactory factory, int m
         // that redefines these constants and is not supposed to run existing programs.
 
         /// <summary>
-        /// Corresponds to <see cref="Microsoft.CSharp.RuntimeBinder.CSharpBinderFlags"/>.
+        /// Corresponds to Microsoft.CSharp.RuntimeBinder.CSharpBinderFlags.
```

Comment:
Tag @jasonmalinowski, but I thought his reasoning was that you should suppress the warning if there was really a reason to use T: (like this one).